web/client/components/map/openlayers/plugins/WMTSLayer.js

```diff
@@ -90,6 +91,19 @@ const createLayer = options => {
         && tileMatrixSet.TileMatrix
             .map(({ TileWidth, TileHeight } = {}) => ([parseInt(TileWidth, 10), parseInt(TileHeight, 10)]));
 
+    const bbox = options.bbox;
```

Comment:
I try, correct me if I'm wrong
```suggestion
    // if the layer comes with bbox, it can be used as extent to define the tile source's extent (and avoid to load tiles out of this area. Otherwise the default extent of the projection will be used.
    const bbox = options.bbox;
```

Reply:
Well. What I was asking for is how to change the code to make It more clear so that we don't need to comment It. I don't like putting comments everywhere except in rate occasions